tsdb/db.go

```diff
@@ -93,31 +101,13 @@ type Options struct {
 
 	// StripeSize is the size in entries of the series hash map. Reducing the size will save memory but impact perfomance.
 	StripeSize int
-}
 
-// Appender allows appending a batch of data. It must be completed with a
```

Comment:
This documentation is lost, existing storage.Appender doesn't have it anymore.

Reply:
Important, thanks!